src/components/header/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import { NavLink } from "react-router-dom";
+import "./header.scss";
+
+export default function Header(){
+  const [show, setShow] = useState('hide');
+  return (
+    <nav className="navbar navbar-expand-lg navbar-dark bg-dark w-100">
+      <NavLink className="navbar-brand" to={"/"}>Ecommerce</NavLink>
+      <button className="navbar-toggler" type="button"  onClick={()=>{show === "hide"?setShow("show"):setShow("hide")}} data-toggle="collapse" data-target="#navbar-menu" aria-controls="navbar-menu" aria-expanded="false" aria-label="Toggle navigation">
+        <span className="navbar-toggler-icon"></span>
+      </button>
+      <div className={"collapse navbar-collapse "+show } id="navbar-menu">
+        <ul className="navbar-nav">
```

Comment:
Have the meaning name for the classes, may be navbar list.

Reply:
this class name are predefined in bootstrap.